pkg/machinery/constants/constants.go

```diff
@@ -881,6 +881,9 @@ const (
 
 	// MaintenanceServiceCommonName is the CN of the maintenance service server certificate.
 	MaintenanceServiceCommonName = "maintenance-service.talos.dev"
+
+	// GRPCMaxMessageSize is the maximum message size for Talos API.
+	GRPCMaxMessageSize = 32 * 1024 * 1024
```

Comment:
do we really want to support a 32 meg config patch :sweat_smile:

Reply:
not that big probably, but bump it up from a default 4 MiB limit. Watch API call returns old/new state, so 32 MiB will be split in a half, plus compression